scripts/build_packages.py

```
@@ -0,0 +1,296 @@
+    print(f"✅ Version: {version}")
+    
+    return output_dir, version
+
+
+def build_packages(dist_dir):
+    """Build wheels and source distributions for both packages."""
+    print("Building packages...")
+    
+    # Build fla-core (both wheel and sdist)
+    print("Building fla-core packages...")
+    result = subprocess.run([
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)